src/content/1.7/scale/optimizations.md

```diff
@@ -4,23 +4,45 @@ alias:
 - 1.7/optimizations/
 ---
 
-# How to Optimize the performance of your PrestaShop
+# Optimize your PrestaShop performances
 
-In order to properly benchmark your shop, you need to check the settings of PHP, Apache and MySQL.
+Before making any change on your PrestaShop instance, it is highly recommended to perform some benchmarks.
 
-## 1) PHP Settings
+The idea behind this is to _know_ where you're coming from and make sure that your changes increase your performances, not the other way around.
 
-Try to use PHP >=7, it will speed up a lot your shop!
+And if you've done no benchmark there is no way to ensure you did not decrease your performances in the end.
 
-If you're using PHP-FPM (which should be the case in most of "modern" installations), you have to check the pool
+{{% notice tip %}}
+Keep in mind that the surest way to tune your shop is:
+
+- benchmark
+- make a change
+- repeat
+{{% /notice %}}
+
+## Finding your bottleneck(s)
+
+Performance tuning can be a tricky thing, there are plenty of reason an application can be slowed down.
+
+It can come from your disks, lack of memory, network capacity, CPU: wherever you can think of.
+
+Then, the challenge is to find your _current_ bottleneck (could be disks access), remove it and then find the next one (could be memory).
+
+There's no way to enumerate all the performance issues we've encountered, but let's review the more common ones and tackle them.
+
+### 1) PHP
+
+First, try to use PHP >=7 when possible. Hard work has been done on performance and it will provide a nice boost to your shop!
```

Comment:
```suggestion
First, try to use PHP >=7 when possible. Hard work has been done on performance starting on this version, and it will provide a nice speed boost to your shop!
```